lib/ansible/modules/cloud/lxd/lxd_network.py

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Sirtaj Singh Kang <sirtaj@sirtaj.net>
+# (c) 2016, Hiroaki Nakamura <hnakamur@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: lxd_network
+short_description: Manage LXD network
+version_added: 2.7
+description:
+  - Management of LXD networks
+author: "Sirtaj Singh Kang (@sirtaj) and Hiroaki Nakamura (@hnakamur)"
```

Comment:
not a blocker but create a list for the authors (keep the key "author" as is)

~~~
- "Sirtaj Singh Kang (@sirtaj) 
- "Hiroaki Nakamura (@hnakamur)"
~~~